File: 02. RFID/Film_SparePart_RFID/Library.Reader/Objects/ReaderEmailObject.cs
using System;

namespace Library.Reader.Objects
{
    public class ReaderEmailObject
    {
        public ReaderEmailObject(string device_name, string ip_address, string location_desc, DateTime time_occured, string message)
        {
            DeviceName = device_name;
            IPAddress = ip_address;
            LocationDesc = location_desc;
            TimeOccured = time_occured;
            Message = message;
        }

        public string Company { get; set; }
        public string DeviceName { get; set; }
        public string IPAddress { get; set; }
        public string LocationDesc { get; set; }
        public DateTime TimeOccured { get; set; }
        public string Message { get; set; }
    }
}

File: 02. RFID/Film_SparePart_RFID/Library.Reader/Objects/ReaderEmailObject.vb
Namespace Objects

    Public Class ReaderEmailObject

        Public Sub New(ByVal device_name As String, ByVal ip_address As String, ByVal location_desc As String, ByVal time_occured As DateTime, ByVal message As String)

            Me.DeviceName = device_name
            Me.IPAddress = ip_address
            Me.LocationDesc = location_desc
            Me.TimeOccured = time_occured
            Me.Message = message

        End Sub

        Public Property Company As String

        Public Property DeviceName As String

        Public Property IPAddress As String

        Public Property LocationDesc As String

        Public Property TimeOccured As DateTime

        Public Property Message As String


    End Class


End Namespace
